src/plugins/auto/plugin_config.cpp

```diff
@@ -5,6 +5,7 @@
 
 namespace MultiDevicePlugin {
 const std::set<std::string> PluginConfig::_availableDevices = {"AUTO", "CPU", "GPU", "TEMPLATE", "NVIDIA", "VPUX", "MULTI", "HETERO", "mock"};
+const std::set<std::string> PluginConfig::_deviceBlacklist = {"VPUX", "GNA"};
```

Comment:
Please add description about blacklist in code

Reply:
Updated.